client/src/services/translation.ts

```diff
@@ -65,3 +65,7 @@ export function getInvitationStatusTranslationKey(status: InvitationStatus): Tra
       return 'UsersPage.invitation.status.cancelled';
   }
 }
+
+export function longLocaleCodeToShort(longCode: string): string {
```

Comment:
To be a bit pedanctic, we technically have the type `Locale` (https://github.com/Scille/megashark-lib/blob/a543a0a0b47bef7b926375b16f29bbdf1b1d5dfc/lib/services/translation.ts#L24), it ensures that the string has the right format.

Reply:
done